build.gradle

```diff
@@ -125,6 +125,34 @@ publishing {
                 }
 
                 developers {
+                    developer {
+                        name = "Civitaspo"
+                        email = "civitaspo@gmail.com"
+                    }
+                    developer {
+                        name = "Satoshi Akama"
+                        email = "satoshiakama@gmail.com"
+                    }
+                    developer {
+                        name = "Naotoshi Seo"
+                        email = "sonots@gmail.com"
+                    }
```

Comment:
@sonots We're nominating you in `pom.xml`'s `<developers>` like we did in https://github.com/embulk/embulk/pull/1399#pullrequestreview-601345451.